docs/mkdocs/docs/index.md

```diff
@@ -339,7 +341,7 @@ VersionedItem(symbol=test_frame,library=data,data=n/a,version=2,metadata=None,ho
 2000-01-02 09:00:00      2      1     20     47     47     16     14     48  ...
 ```
 
-The final 7 rows included the 5 rows we have just appended and the last two rows that were written previously. 
+The final 7 rows consist of the last two rows written previously followed by the 5 rows that we have just appended.
```

Comment:
Could you check that this code example is really how ArcticDB works? Probably adding a unit test for it is good idea if one doesn't already exist.

I think this example, with the duplicate in the index, is a bit confusing and might have been a typo. I think our first append example should do what it originally and falsely promised to do:

```
 Note the starting date of this DataFrame is after the final row written previously!
```

Reply:
I've changed the docs example to append data that starts after the final row written previously, and also added a unit test to verify this. The unit test also prints out the outputs for the other examples so that it is easier if somebody in the future wants to modify something and re-generate all the outputs.